components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -275,7 +275,8 @@ public sealed class PublisherJobService : IPublishServices<string> {
                             BatchTriggerInterval = DefaultBatchTriggerInterval.Value,
                             DiagnosticsInterval = TimeSpan.FromSeconds(60),
                             MaxMessageSize = 0,
-                            MaxOutgressMessages = DefaultMaxOutgressMessages.Value
+                            MaxOutgressMessages = DefaultMaxOutgressMessages.Value,
+                            HasRoutingInfo = false,
                     };
```

Comment:
Seems like this `};` is missing one additional level of indentation.

Reply:
done